### ISSUE TITLE
Research, discuss and re-implement useful Athenaeum features

### ISSUE THREAD

**Issue by neuromancer** [tkashkin/GameHub#62]
The main features of Athenaeum are explained [here](https://gitlab.com/librebob/athenaeum). In particular, I think we can discuss:

* Flatpak download and installation 
* License filtering

**Comment by tkashkin** [comment]
So this thing loads anything with Game category from flathub, right?
It shouldn't be too hard to implement.

Does PackageKit support flatpak? It may be better, because it will allow games from distro repos too.

By "license filtering" you mean automatic tagging?

**Comment by neuromancer** [comment]
If someone wants to download and play only free/open source games, a good idea is to allow to filter flatpaks game by their license.

**Comment by Saroufim** [comment]
It would also be nice to add Winepak. GameHub itself is better off not flatpaked due to sandboxing issues. However, it could become an excellent manager for flatpaked games.

**Comment by neuromancer** [comment]
I agree it could be nice, but winepack seems to require to [build wine (and dependencies)](https://github.com/winepak/winepak-sdk-images#building-the-runtime), and it can take a while..

**Comment by emorrp1** [comment]
> Does PackageKit support flatpak? It may be better, because it will allow games from distro repos too.

PackageKit is [defunct] since 2014. In that post they basically say only two backends (dnf and apt) were needed, so the extremely generic abstractions weren't used. I would very much like support for native repos, but it seems to me that could be prototyped with Flathub and later have a very thin layer to add dnf and apt as alternatives under the same "native store" button.

[defunct]: https://blogs.gnome.org/hughsie/2019/02/14/packagekit-is-dead-long-live-well-something-else/
